### PR TITLE
HTML file for showing interface for uploading files was added. Wrote …

### PR DESCRIPTION
**Pull request by meharoofnajeeb**
…a post method for getting file uploads to uploads folder in project folder in index.js file. Added utility methods in utils folder and separated them into respective files. Removed views folder (jade files).
@abhilashsajeev @AmalQB @madhukarmmallia

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,12 +1,51 @@
...
+  var nameSplitArray = _.split(req.file.originalname, '.', 10);
+  var extension = nameSplitArray[nameSplitArray.length - 1];
+  if (extension !== 'xlsx' && extension !== 'csv') {
+    fs.unlink('uploads/' + storedFileName, function (err) {
+      if (err) {
+        return console.log(err);
+      }
+      console.log('File deleted successfully');
+    });
+    res.sendStatus('Unsupported file type');
```
Comment (abhilashsajeev) [inline routes/index.js]: @meharoofnajeeb 
express api says to send a status code. Please clarify.
http://expressjs.com/en/api.html#res.sendStatus

res.sendStatus(statusCode)

File: routes/index.js [hunk 2]
```diff
@@ -1,12 +1,51 @@
...
+  }
+  console.log('parseFile method is gonna get called');
+  xlsxUtil.parseFile(storedFileName);
+  res.sendStatus('Post file finished');
```
Comment (abhilashsajeev) [inline routes/index.js]: @meharoofnajeeb Same as above comment regarding status code.

**Comment by meharoofnajeeb** [comment]
@abhilashsajeev

**Comment by abhilashsajeev** [comment]
@madhukarmmallia @sam-qburst @AmalQB Looks good to me. Merging

**Pull request merged**